app/models/creator.rb

```diff
@@ -0,0 +1,115 @@
+class Creator < ActiveRecord::Base
+  has_many :content
+  has_and_belongs_to_many :users
+  validates :p_id, :provider, :username, presence: true
+
+  include HTTParty
+  LIMIT_PER_PAGE = 10
+
+  #method to configure twitter
+  def twit
+    Seemore::Application.config.twitter
+  end
+
+  def has_image?
+    if avatar_url.nil?
+      false
+    else
+      true
+    end
```

Comment:
You could use a ternary operator here:
` avatar_url.nil? ? false : true`
Or if you want to re-write it so that the true-condition is true (if that's not confusing, I don't know what is):
`avatar_url ? true : false`
Even if you don't want to use the ternary operator, I prefer leaving off the `.nil?`, so I'd write it as:
```
if avatar_url
  true
else
  end
end
```